spec/functions/deprecation_spec.rb

```diff
@@ -42,6 +42,7 @@
   end
 else
   describe 'deprecation' do
+    ENV['STDLIB_LOG_DEPRECATIONS'] = "true"
```

Comment:
After discussion David and I decided the code change to add before hooks and contexts is too much for the simplicity of the change. Instead I will add after hooks to each of the test for cleanup.